x-pack/legacy/plugins/siem/public/components/url_state/helpers.ts

```diff
@@ -113,42 +108,13 @@ export const getTitle = (
   return navTabs[pageName] != null ? navTabs[pageName].name : '';
 };
 
-export const getCurrentLocation = (
```

Comment:
FYI -> It was not used @stephmilovic told me and when I went in this files, I had to delete it ;)